### PR TITLE
P0 filter tags

### PR DESCRIPTION
**Pull request by dev-prabhat**
Check these files 
https://github.com/dev-prabhat/note-it-down/pull/11/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&show-viewed-files=true

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI 
Good semantic tags
Good for approval ✅

**Review by srishtigautam21 — APPROVED** [review]
The code is well written
Good work prabhat
Good to merge

File: src/components/EditForm/EditForm.js [hunk 1]
```diff
@@ -28,26 +40,39 @@ export const EditForm = ({setEditModal}) => {
...
-                <RichTextEditor className="padding-xs" setRichTextEditor={setEditNote} richTextContent={editNote}/>
+                <AiOutlineClose 
+                    title="close" 
+                    className="close-Icon icons-common" 
+                    onClick={()=>setEditModal(prev => !prev)}
+                />
+                <RichTextEditor 
+                    className="padding-xs" 
+                    setRichTextEditor={setEditNote} 
+                    richTextContent={editNote}
```
Comment (Dev-AkshatGupta) [inline src/components/EditForm/EditForm.js:51]: Nice that you are using a rich text editor

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything  looks good to me.
You can merge it

**Pull request merged**